src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -38,10 +38,10 @@
 /**
  * The {@link BlockListener} is responsible for listening to the {@link BlockPlaceEvent}
  * and {@link BlockBreakEvent}.
- * 
+ *
  * @author TheBusyBiscuit
  * @author Linox
- * 
```

Comment:
Speaking of which, since you modified this class, feel free to add yourself here.
So we know who to contact XD